### PR TITLE
DBaaS-95&99: Update dbaas-operator to be cluster-scoped, hide DBaaSConnection from OLM

### PR DESCRIPTION
**Pull request by xieshenzh**
## Description

1. hide DBaaSConnection from OLM: https://issues.redhat.com/browse/DBAAS-99
2. update dbaas-operator to be cluster-scoped: https://issues.redhat.com/browse/DBAAS-95

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA or in issue number have been completed
- [ ] Unit tests added that prove the fix is effective or the feature works 
- [ ] Documentation added for the feature
- [ ] all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer

### CODE REVIEW

File: PROJECT [hunk 1]
```diff
@@ -9,7 +9,7 @@ repo: github.com/RHEcosystemAppEng/dbaas-operator
 resources:
 - api:
     crdVersion: v1
-    namespaced: true
+    namespaced: false
```
Comment (tchughesiv) [inline PROJECT:12]: the CRDs themselves should stay namespace scoped. CRD scoping is completely separate from Operator scoping. This should remain true.

File: PROJECT [hunk 2]
```diff
@@ -18,7 +18,7 @@ resources:
   version: v1alpha1
 - api:
     crdVersion: v1
-    namespaced: true
+    namespaced: false
```
Comment (tchughesiv) [inline PROJECT:21]: the CRDs themselves should stay namespace scoped. CRD scoping is completely separate from Operator scoping. This should remain true.

**Review by tchughesiv — CHANGES_REQUESTED** [review]

File: PROJECT (hunk 1 above)
Comment (tchughesiv) [reply PROJECT:12]: this is important so that developers can set connections within their namespaces.

File: PROJECT (hunk 2 above)
Comment (tchughesiv) [reply PROJECT:21]: this is important so that tetant service admins can set inventory specific to a given tenant.

**Review by jeremyary — APPROVED** [review]

**Pull request merged**